src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -239,7 +245,7 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
     new JesBackendCall(this, workflowDescriptor, key, locallyQualifiedInputs, abortRegistrationFunction)
   }
 
-  override def engineFunctions: WdlStandardLibraryFunctions = new JesEngineFunctionsWithoutCallContext()
+  override def engineFunctions(interface: IOInterface): WdlStandardLibraryFunctions = new JesEngineFunctionsWithoutCallContext(interface)
```

Comment:
Added a `type IOInterface` as per @geoffjentry suggestion that is overriden by the backend implementation (similarly to BackendCall), so that passing the wrong IOInterface implementation won't compile.